app/javascript/components/mentoring/inbox/TrackList.tsx

```diff
@@ -5,15 +5,15 @@ export type Track = {
   slug: string
   title: string
   iconUrl: string
-  count: number
+  count?: number
 }
 
 const OptionComponent = ({ option: track }: { option: Track }) => {
   return (
     <React.Fragment>
       <TrackLogo track={track} />
       <div className="title">{track.title}</div>
-      <div className="count">{track.count}</div>
+      {track.count && <div className="count">{track.count}</div>}
```

Comment:
Why this change? And should it be a different PR (as it's to do with mentoring inbox)?

Reply:
CC @dem4ron